pkg/assembler/backends/ent/schema/dependency.go

```diff
@@ -63,6 +63,7 @@ func (Dependency) Edges() []ent.Edge {
 			Field("package_id").
 			Unique().Annotations(entsql.OnDelete(entsql.Cascade)),
 		edge.To("dependent_package_version", PackageVersion.Type).
+			Required().
```

Comment:
I have created a script located here: https://github.com/pxp928/guac-update-db

That can be run so that the migration can occur without an issue.

This is a breaking change to exisiting ENT databases.